src/teams.rs

```diff
@@ -98,22 +155,23 @@ fn read_rfcbot_cfg() -> RfcbotConfig {
 }
 
 fn read_rfcbot_cfg_from(input: &str) -> RfcbotConfig {
-    toml::from_str(input).expect("couldn't parse rfcbot.toml!")
+    let mut config: RfcbotConfig = toml::from_str(input).expect("couldn't parse rfcbot.toml!");
+    config.update().expect("couldn't update the configuration!");
```

Comment:
`read_rfcbot_cfg_from` is meant to be a "pure" function for testing purposes; move call to update to `read_rfcbot_cfg` instead?